modules/scope-common-mappers/main.tf

```diff
@@ -107,6 +114,7 @@ resource "keycloak_generic_protocol_mapper" "claim_omitter" {
   protocol        = "openid-connect"
   protocol_mapper = "omit-claim-by-idp-mapper"
   config = {
+    "introspection.token.claim" : "true",
```

Comment:
Does claim omitter have the introspection option? I only see the userinfo in the ui

Reply:
It seems to be a part of  the  mapper
```
  ~ resource "keycloak_generic_protocol_mapper" "claim_omitter" {
      ~ config          = {
          + "introspection.token.claim" = "true"
```